src/ImageMetadata.jl

```diff
@@ -311,8 +317,8 @@ function permutedims(img::ImageMeta, perm)
     permutedims_props!(copyproperties(img, permutedims(img.data, perm)), ip)
 end
 
-Base.ctranspose(img::ImageMeta{T,2}) where {T<:Real} = permutedims(img, (2,1))
-function Base.ctranspose(img::ImageMeta{T,1}) where T<:Real
+Base.adjoint(img::ImageMeta{T,2}) where {T<:Real} = permutedims(img, (2,1))
```

Comment:
Not really relevant for this PR, but this brings up a question: 

Should we deprecate the use of `adjoint` for `ImageMeta`? mapping to `permutedums` looks like we are abusing it a bit

Reply:
Or map to `adjoint`. I think when this was written we didn't have a non-copying `adjoint` in Base.

Reply:
The docs say `adjoint` is "intended for linear algebra usage" and recursive, so what if some properties are array-like?  @Evizero's suggestion seems safest.